### ISSUE TITLE
Failed to import configuration with ECC-384 Algorithm

### ISSUE THREAD

**Issue by LNono7** [OpenVPN/openvpn3-indicator#16]
Hi,
I'm having a problem importing an OpenVPN configuration using the ECC-384 algorithm on Ubuntu 22.04.4 with openvpn3-indicator. The import fails with the following error message :
`net.openvpn.openvpn3_indicator.desktop[7694]: ERROR:root:Failed to import configuration NEW from ...
`
When I manually run with openvpn3 session-start my OpenVPN profile, I get an error with the application launched with the following error :
`session-start: ** ERROR ** Failed to start new session: Failed calling D-Bus method [net.openvpn.v3.sessions] net.openvpn.v3.sessions.Connect: Timeout was reached
`
But when the application is not launched, it works fine.

**Comment by titov-vv** [comment]
Err... I looks a bit similar to mine #15 . Have your tried it with log level DEBUG to see more detailed exception?
With regards to `openvpn3 session-start` - have you tried to start it several times? I recently got a situation when first execution fails with similar error but then 2nd or 3rd execution establishes a tunnel (I'm not on Ubuntu, so might be different problem, but for me it was caused by some temporary package incompatibility that was resolved with next upgrade).
